dev_guide/managing_images.adoc

```diff
@@ -1099,6 +1099,62 @@ my-image   172.30.48.125:5000/test/my-image   latest    3 seconds ago
 ----
 ====
 
+
+[[triggering-updates-on-image-stream-changes]]
+== Triggering Updates On Image Stream Changes
+
+=== Overview
+When an image stream tag is updated to point to a new image, {product-title} can automatically take action to roll the new image out to resources that were using the old image.  This is configured in different ways depending on the type of resource that is referencing the image stream tag.
+
+=== OpenShift Resources
+OpenShift DeploymentConfigs and BuildConfigs can be automatically triggered by changes to ImageStreamTags.  The triggered action can be run using the new value of the image referenced by the updated ImageStreamTag.  For more details on using this capability see the documentation on xref:builds/triggering_builds.adoc#image-change-triggers[BuildConfig triggers] and xref:deployments/basic_deployment_operations.adoc#image-change-trigger[DeploymentConfig triggers].
+
+=== Kubernetes Resources (Alpha)
+Unlike DeploymentConfigs and BuildConfigs, which include as part of their API definition a set of fields for controlling triggers, Kubernetes resources don't have fields for triggering. Instead, {product-title} uses annotations to allow users to request triggering. The annotation is currently alpha and is defined as follows:
+
+====
+----
+Key: image.alpha.openshift.io/triggers
+Value: array of triggers, where each item has the schema:
+[
+ {
+   "from" :{
+     "kind": "ImageStreamTag", // required, the resource to trigger from, must be ImageStreamTag
+     "name": "example:latest", // required, the name of an ImageStreamTag
+     "namespace": "myapp",     // optional, defaults to the namespace of the object
+   },
+   // required, JSON path to change
+   // Note that this field is limited today, and only accepts a very specific set
+   // of inputs (a JSON path expression that precisely matches a container by ID or index).
+   // For pods this would be "spec.containers[?(@.name='web')].image".
+   "fieldPath": "spec.template.spec.containers[?(@.name='web')].image"
```

Comment:
it is missing "paused" field which indicates if this trigger is on/off.

Reply:
thanks, will add.